recipe/recipe.yaml

```diff
@@ -1,18 +1,22 @@
 context:
   name: "fastjet-cxx"
-  version: "3.4.3"
+  version: "3.5.0"
 
 package:
   name: ${{ name }}
   version: ${{ version }}
 
 source:
   url: https://gitlab.com/fastjet/fastjet/-/archive/fastjet-${{ version }}/fastjet-fastjet-${{ version }}.tar.gz
-  sha256: 8ce8b9be5cd0fba4f2fe914871eabe259dfa7524edf925635e20072cde2ae6a8
-  patches: 0001-use-conda-forge-siscone-for-plugin.patch
+  sha256: 67c314aec7899f098c9ec8fb2ca0c46a3b7700cbb756278ab4a6a02f34ba1dc9
 
 build:
-  number: 2
+  number: 0
+  script:
+    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DFASTJET_ENABLE_ALLCXXPLUGINS=ON -DFASTJET_ENABLE_CGAL=ON -DFASTJET_ENABLE_PYTHON=ON -DFASTJET_USE_INSTALLED_SISCONE=ON -DFASTJET_PYTHON_PACKAGE_NAME="fastjet_cxx" -DPython_EXECUTABLE=$(which python) -S . -B build
```

Comment:
```suggestion
    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DFASTJET_ENABLE_ALLCXXPLUGINS=ON -DFASTJET_ENABLE_CGAL=ON -DFASTJET_ENABLE_PYTHON=ON -DFASTJET_USE_INSTALLED_SISCONE=ON -DFASTJET_PYTHON_PACKAGE_NAME="fastjet_cxx" -DPython_EXECUTABLE=$PYTHON -S . -B build
```

Reply:
`$PYTHON` should resolve correctly in `rattler-build`.

Reply:
Ah yeah, good catch @lgray. `${{ PYTHON }}` is needed in the `recipe.yaml` file and you can only use `$PYTHON` in a build script. Duh. My bad.